index.html

```diff
@@ -1576,7 +1574,7 @@ <h4>
         USVString encoding;
         USVString lang;
 
-        NDEFRecordDataSource data;
+        any data;
```

Comment:
Maybe for documentation purposes like MDN we should add a note that the accepted types are actually DOMString, BufferSource and NDEFMessageInit and that currently all other values will be ignored?

Reply:
I've added a comment.